cam0:
  T_cam_imu:
  - [ 0.99975613, -0.01445535,  0.01669491, 0.01821261]
  - [-0.01524656, -0.99871745,  0.04828046, -0.0897189 ]
  - [ 0.01597559, -0.04852323, -0.99869429, 0.03]
  - [0.0, 0.0, 0.0, 1.0]
  cam_overlaps: [1,2]
  camera_model: pinhole
  distortion_coeffs: [0.0744377830900075, -0.6024836728067258, -0.0007612691127689372, -0.0026204638007033187]
  distortion_model: radtan
  intrinsics: [4244.232895576106, 4241.6422207651085, 928.0849188317557, 578.3870040527507]
  resolution: [1920, 1080]
  rostopic: /cam0/rgb_cam/image_raw
imu:
  accelerometer_noise_density: 0.00023924597133740611 
  accelerometer_random_walk: 2.3495272461355257e-05 
  gyroscope_noise_density: 0.00011202061515806617 
  gyroscope_random_walk: 4.441401952775738e-06 
  rostopic: '/imu/data'  # Make sure this is correct
  update_rate: 100.0  # Make sure this is correct
ublox:
  T_ubl_imu:
  - [1.0, 0.0, 0.0, -0.015]
  - [0.0, 1.0, 0.0, 0.0]
  - [0.0, 0.0, 1.0, 0.33]
  - [0.0, 0.0, 0.0, 1.0]
  
